src/SearchBar.jsx

```diff
@@ -6,8 +7,17 @@ const capitalPDangit = (query) => {
   return query.replace(/Wordpress/i, "WordPress");
 };
 
+const types = {
```

Comment:
Can we reword this a little bit:
- WordPress reference
- WordPress dev notes
- WP-CLI
- PHP reference

Reply:
Done in https://github.com/zzap/docs-dangit/pull/21/commits/92b1b3c11f2b00f68b7daace7b89486071e16dcd